crates/walrus-service/src/node/event_blob_writer.rs

```diff
@@ -1439,7 +1456,11 @@ impl EventBlobWriter {
         let blob_id = *blob_metadata.blob_id();
 
         let mut batch = self.pending.batch();
-        match self.attest_blob(&blob_metadata, metadata.end).await {
+        let active_committees = self.node.committee_service.active_committees();
+        match self
+            .attest_blob(&blob_metadata, metadata.end, &active_committees)
```

Comment:
question: it is possible that this node is lagging, and in the event blob's epoch, the node is not part of the committee, but self.node.committee_service.active_committees(), which returns the latest epoch (can be far ahead than event blob epoch), has the node in the committee. Is it ok if the node attests the blob in this case?